gpu-usage

```diff
@@ -8,37 +8,47 @@ print_usage () {
 Usage: $0 [-n charles[01-19]]
     Get gpu usage for all, or specified nodes
 
-Arguments
+Arguments:
     -n    (optional) specify nodes, either comma separated list, or summarised.
           Mirrors slurm command -n usage in sinfo, and -w usage for squeue." 
     -h    (optional) do not print header
     -p    (optional) make output pretty, ignores -h if specified
 
+Output:
+    Outputs three columns summarising the current usage of GPUs on the cluster:
+        -- The 'in_use' column shows the number of GPUs that are currently allocated to a job.
+        -- The 'usable' column shows the number of GPUs that are not in an error state or otherwise unable to be
```

Comment:
You know... at some point we should change this command to actually return the number of free gpus there are available! Currently you have to do usable - in_use in your head (not a big ask, but doesn't make it entirely clear).

Reply:
Yeah, I was thinking that when I was writing the documentation. I'll maybe look at it later today -- I guess it shouldn't be too complicated to do.

Reply:
Def don't prioritise. If you do do, please may we retain the current columns, just so as it won't break my logging scripts?